tests/testthat/test-navbar.R

```diff
@@ -7,6 +7,9 @@ test_that("adds github/gitlab link when available", {
 
   pkg <- pkg_navbar(github_url = "https://gitlab.com/r-lib/pkgdown")
   expect_snapshot_output(navbar_components(pkg))
+
+  pkg <- pkg_navbar(github_url = "https://codeberg.org/r-lib/pkgdown")
```

Comment:
I don't tihnk it's worth adding a snapshot test here as your code is already covered by the other tests, and snapshot tests are expensive to review.